server/apps/main/forms.py

```diff
@@ -77,6 +77,21 @@ class ShareExperienceForm(forms.Form):
     authorship_relation.layout_horizontal = True
     authorship_relation.group = 4
 
+
+    def clean(self):
+        # Raise an error if second-hand authorship chosen with no description
+        # Raise an error if first-hand authorship has a relationship description
+
+        cleaned_data = super().clean()
+
+        first_hand_authorship = cleaned_data.get("first_hand_authorship")
+        authorship_relation = cleaned_data.get("authorship_relation")
+
+        if first_hand_authorship == "False" and authorship_relation == "":
+                self.add_error("first_hand_authorship", ValidationError(("Stories written on behalf of someone else must have a description of the relationship"), code="invalid"))
+        if first_hand_authorship == "True" and authorship_relation != "":
```

Comment:
From a brief look around the docs it might not be straightforward as it depends on two fields so those checks happen after the individual field form validation but I will give it a shot - I guess plan B would be a more verbose error message?

Reply:
Yeah, that's fair! Alternatively we could do that "removal" of the entered text using JS in the front-end on submission?